packages/bento-design-system/src/SelectField/createSelectField.tsx

```diff
@@ -174,7 +176,7 @@ export function createSelectField(
           hideSelectedOptions={false}
           menuSize={menuSize}
           menuIsOpen={isReadOnly ? false : undefined}
-          isSearchable={isReadOnly ? false : undefined}
+          isSearchable={isReadOnly ? false : searchable ?? false}
```

Comment:
```suggestion
          isSearchable={isReadOnly ? false : searchable ?? true}
```

We discussed this internally, and let's keep it `true` by default so to avoid a breaking change